src/quo2/components/list_items/dapp/style.cljs

```diff
@@ -0,0 +1,41 @@
+(ns quo2.components.list-items.dapp.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container-outer
+  [{:keys [state blur? customization-color theme]}]
+  {:flex               1
+   :padding-horizontal 12
+   :padding-vertical   8
+   :border-radius      12
+   :background-color   (cond
```

Comment:
I like putting long conditions like this into their own function.

Reply:
yes, me too! done.